### PR TITLE
Disable weekly updates

### PR DESCRIPTION
**Pull request by BryonLewis**
This leaves the watchtower container running but prevents it from doing the weekly update on the girder/girder-worker containers.  This was request to prevent it from updating weekly like this.

https://containrrr.dev/watchtower/arguments/#filter_by_enable_label

That should provide some information on why I commented out the label in the docker-compose.yml

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ services:
       - "RABBITMQ_MANAGEMENT_URL=${RABBITMQ_MANAGEMENT_URL:-http://rabbit:15672/}"
       - "RABBITMQ_MANAGEMENT_BROKER_URL_TEMPLATE=${RABBITMQ_MANAGEMENT_BROKER_URL_TEMPLATE}"
     labels:
-      - "com.centurylinklabs.watchtower.enable=true"
+      # - "com.centurylinklabs.watchtower.enable=true" REMOVING WEEKLY UPDATE
```
Comment (waxlamp) [inline docker-compose.yml:88]:
```suggestion
      # REMOVING WEEKLY UPDATE
      # - "com.centurylinklabs.watchtower.enable=true"
```

**Review by waxlamp — CHANGES_REQUESTED** [review]
Just to make it a bit easier to properly uncomment this in the future.

**Review by marySalvi — COMMENTED** [review]
Should this be a merge from main or is it better to rebase here?

**Comment by BryonLewis** [comment]
> Should this be a merge from main or is it better to rebase here?

I don't think it matters as it gets into main because it will be a squash anyways, so only the changes get committed into main.  So in the overall git history it will look like a single commit.

If you take a look into the PR for 1.9 and activity groups getting added in there are lot of merges of main into the branch, but the history only has a single commit with the changes.  It means we don't have to be exact with our development process or clean up the git history before we merge the PR, but we still retain a history of what occurred within the individual PR.  There are probably good arguments for doing it the opposite as well.
Activity Groups PR: https://github.com/Kitware/dive/pull/1247

**Review by marySalvi — APPROVED** [review]

**Review by waxlamp — APPROVED** [review]

**Pull request merged**